Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace MathFormula
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Please, write x: ");
+            double x = Convert.ToDouble(Console.ReadLine());
+
+            double sin = Math.Sin(Math.Sqrt(Math.Pow(x, 2) + 4) / 2);
+            double cos = Math.Cos(Math.Sqrt(Math.Pow(x, 2) + 1) / 2); 
+
+            Console.WriteLine("Answer: {0}", Math.Round(2 * sin * cos, 3)); 

```

Comment:
В задании не требовалось выводить вычисленное значение с точностью до 3-х знаков после занятой.
```suggestion
            Console.WriteLine("Answer: {0}", 2 * sin * cos); 

```